virttest/utils_v2v.py

```diff
@@ -714,7 +715,12 @@ def vm_journal_search(self, substr, options=None, flags=re.IGNORECASE):
         if options:
             cmd += " %s" % options
 
-        if self.vm_general_search(cmd, substr, flags, ignore_status=True, debug=False):
+        if self.vm_general_search(
+                cmd,
+                substr,
+                flags,
+                ignore_status=True,
+                debug=False):
```

Comment:
I don't think we should mix the change of code format with the change of function. And to be honest, I don't like this kind of format.

Reply:
I see, but this really can save your energy on modifying the format. and it's fully obey the pep8.
when your have a small monitor or  you read the code in vim, a long line will be warped, then it's different to read.
I will not mix those format changes with functional codes in future. Thanks.